.github/workflows/rust.yml

```diff
@@ -116,7 +116,3 @@ jobs:
       - name: Deploy develop # unused
         if: github.ref == 'refs/heads/develop'
         run: bash scripts/deploy_docker.sh develop
```

Comment:
This can also be removed right?

Reply:
I think I would keep it, as it is not entirely unlikely we introduce a dev branch at some point